### ISSUE TITLE
Code state/documentation

### ISSUE THREAD

**Issue by Supersak80** [kuangliu/pytorch-retinanet#3]
Hi @kuangliu, does this code follow the paper exactly? Also, are you able to document it sufficiently so that we can get started on it using VOC?

**Comment by kuangliu** [comment]
I'm still working on the training code. The whole process "works" now, but it's not fully trained and tested yet.  And I'll update the doc as soon as I get a decent accuracy.
